src/controllers/albumController.js

```diff
@@ -24,6 +55,18 @@ const getSingleAlbum = async (req, res) => {
 };
 
 const updateAlbum = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch an album  by id and update it'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the album to fetch',
+     }
```

Comment:
In addition to the `id` path parameter, we also have the body parameter that the caller needs to pass in. Swagger-autogen doesn't know exactly what that would look like based on the code so we do need to explicitly define that here. We can probably use the same definition that we will create in `swagger.js` for the createAlbum endpoint since I assume the same values are allowed here for the update?

```
     #swagger.parameters['album properties to update'] = {
      in: 'body',
      description: 'Album information to use for update',
      required: true,
      type: 'object',
      schema: { $ref: '#/definitions/NewAlbum' }
     }
```